docs/explanation/development-process.rst

```diff
@@ -1,17 +1,322 @@
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Some translation data cannot currently be updated via the language pack
+mechanism. Because these items require more disruptive integration work,
+they are subject to an earlier deadline to give time to developers to manually
+export translations from Launchpad and integrate them into the package.
+
+This deadline marks the date after which translations for such packages are not
+guaranteed to be included in the final release. Depending on the package and
+its maintainers workflow, they may be exported later.
+
+Other packages can still be translated until the
+:ref:`LanguagePackTranslationDeadline`.
+
+.. _FinalFreeze:
+
+Final Freeze
+~~~~~~~~~~~~
+
+This freeze marks an **extremely** high-caution period until the
+:ref:`FinalRelease`. Only bug fixes for release-critical, security-critical or
+otherwise exceptional circumstantial bugs are included in the Final Release,
+which the release team and relevant section teams must confirm.
+
+Unseeded packages
+^^^^^^^^^^^^^^^^^
+
+Packages in :ref:`ArchiveComponents_Universe` that aren't seeded in any of the
+Ubuntu flavours just remain in :ref:`FeatureFreeze` because they do not affect
+the release; however, when the Ubuntu package archive is frozen, fixes must be
+manually reviewed and accepted by the release team members.
+
+When the Final Release is close (~1.5 days out), developers should consider
+uploading to the :ref:`proposed pocket <ArchivePockets_Proposed>`, from which
+the release team will cherry-pick into the
+:ref:`release pocket <ArchivePockets_Release>` if circumstances allow.
+All packages uploaded to the proposed pocket that do not make it into the
+release pocket until the Final Release will become candidates for
+:ref:`StableReleaseUpdates_Summary`. Therefore, uploads to the proposed pocket
+during Final Freeze should meet the requirements of Stable Release Updates if
+the upload is not accepted into the release pocket. In particular, the upload
+must reference at least one bug, which will be used to track the stable update. 
+
+If you are sure that your upload will be accepted during Final Freeze, you can
+upload directly to the release pocket, but be aware that you have to re-upload
+after Final Release if the upload gets rejected.
+
+.. _ReleaseCandidate:
+
+Release Candidate
+~~~~~~~~~~~~~~~~~
+
+The images produced during the week before the :ref:`FinalRelease` are
+considered "release candidates". In an ideal world, the first release candidate
+would end up being the Final Release; however, we don't live in a perfect
+world, and this week is used to get rid of the last release-critical bugs and
+do as much testing as possible. Until the Final Release, changes are only
+permitted at the release team's discretion and will only be allowed for
+high-priority bugs that might justify delaying the release.
+
+.. _LanguagePackTranslationDeadline:
+
+Language pack translation deadline
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
```suggestion
Language pack translation last day
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
note: I agree that the word `deadline` should be avoided for obvious reasons, but it is the term that was and is used in Ubuntu release schedules (see for example the [Oracular Oriole Release Schedule](https://discourse.ubuntu.com/t/oracular-oriole-release-schedule/36460) or [Language Pack Translation Deadline (Ubuntu Wiki)](https://wiki.ubuntu.com/LanguagePackTranslationDeadline)).

thought: Maybe it is worth having a discussion with the release team to change the used terminology.

Reply:
Thanks for the context. Please, consider the deadline-related suggestion non-blocking.